src/ios/CDVInAppBrowser.m

```diff
@@ -539,115 +545,212 @@ -(void)dealloc {
 - (void)createViews
 {
     // We create the views in code for primarily for ease of upgrades and not requiring an external .xib to be included
+    UIBarButtonItem* flexibleSpaceButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFlexibleSpace target:nil action:nil];
+    UIBarButtonItem* fixedSpaceButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFixedSpace target:nil action:nil];
+    fixedSpaceButton.width = 20;
 
-    CGRect webViewBounds = self.view.bounds;
-    BOOL toolbarIsAtBottom = ![_browserOptions.toolbarposition isEqualToString:kInAppBrowserToolbarBarPositionTop];
-    webViewBounds.size.height -= _browserOptions.location ? FOOTER_HEIGHT : TOOLBAR_HEIGHT;
-    self.webView = [[UIWebView alloc] initWithFrame:webViewBounds];
+    self.webView = [self buildWebView];
+    self.spinner = [self buildSpinner];
+    self.toolbar = [self buildToolbar];
+
+    self.addressLabel = [self buildAddressLabel];
+    self.addressBar = [self buildAddressBar:self.addressLabel];
 
-    self.webView.autoresizingMask = (UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight);
+    self.closeButton = [self buildCloseButton];
+    self.forwardButton = [self buildForwardButton];
+    self.backButton = [self buildBackButton];
+
+    [self.toolbar setItems:@[self.closeButton, flexibleSpaceButton, self.backButton, fixedSpaceButton, self.forwardButton]];
+
+    self.view.backgroundColor = [UIColor whiteColor];
+
+    if (IsAtLeastiOSVersion(@"7.0")) {
```

Comment:
Any reason we should worry about older iOS versions anymore? The latest iOS platform release cordova-ios@4.4.0 only supports iOS 9 and up. 

To support older iOS versions (so users can fall back to older plugin versions for their platform version), you can add engine tag support like so: 
1. https://github.com/phonegap/phonegap-plugin-push/blob/464636b9ae257afc712cfebcc1ed11425590c509/plugin.xml#L14-L18
2. https://github.com/phonegap/phonegap-plugin-push/blob/464636b9ae257afc712cfebcc1ed11425590c509/package.json#L33-L51

Reply:
I only threw in this `IsAtLeastiOSVersion` check because there are other instances of it in this file going all the way back to iOS 6. I can remove it from this commit if you'd like, but I feel like removing the rest of the iOS version checks from this file is probably out of scope of this work.